tutorials/nginx-ingress-gke/index.md

```diff
@@ -0,0 +1,350 @@
+# Ingress with Nginx controller on Google Kubernetes Engine
+
+This guide explains how to deploy the [Nginx Ingress
+Controller](https://github.com/kubernetes/ingress-nginx) on Google Kubernetes
+Engine. 
+
+In Kubernetes,
+[Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/)
+allows external users and client applications access to HTTP services.  Ingress
+consists of two components. 
+[Ingress Resource](https://kubernetes.io/docs/concepts/services-networking/ingress/#the-ingress-resource)
+is a collection of rules for the inbound traffic to reach Services.  These are
+Layer 7 (L7) rules that allow hostnames (and optionally paths) to be directed to
+specific Services in Kubernetes.  The second component is the
+[Ingress Controller](https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-controllers)
+which acts upon the rules set by the Ingress Resource, typically via an HTTP or
+L7 load balancer.  It is vital that both pieces are properly configured to route
+traffic from an  outside client to a Kubernetes Service.  
+
+Nginx is a popular choice for an Ingress Controller for a variety of features:
```

Comment:
NGINX

Reply:
Are you asking to change all instances of Nginx to "NGINX" all caps?

Reply:
This is incorporated